core/src/decl/Kokkos_Declare_SYCL.hpp

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
I'm pretty sure we want these files to have headers and scope guards

Reply:
We don't do this for the respective files for the other backends. :wink:
Let me put a pull request for the other backends first then.

Reply:
See #3453.